util/vendor.py

```diff
@@ -17,15 +17,15 @@
 
 import hjson
 
-DESC = """vendor_hw, copy hardware source code from upstream into this repository"""
+DESC = """vendor, copy source code from upstream into this repository"""
 
 EXCLUDE_ALWAYS = ['.git']
 
 LOCK_FILE_HEADER = """// Copyright lowRISC contributors.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-// This file is generated by the vendor_hw script. Please do not modify it
+// This file is generated by the util/vendor script. Please do not modify it
```

Comment:
While we're changing this, "util/vendor.py script" is perhaps a better description?